providers/gce/gce_loadbalancer_internal.go

```diff
@@ -254,6 +256,37 @@ func (g *Cloud) ensureInternalLoadBalancer(clusterName, clusterID string, svc *v
 	return status, nil
 }
 
+func filterOutNodesFromNonDefaultNetworks(nodes []*v1.Node, defaultSubnetURL string) []*v1.Node {
+	var newList []*v1.Node
+	for _, node := range nodes {
+		subnetLabel, ok := node.Labels[labelGKESubnetworkName]
+
+		if !ok && node.Spec.PodCIDR == "" {
+			continue
+		}
+		if ok && subnetLabel != "" && subnetLabel != subnetNameFromURL(defaultSubnetURL) {
```

Comment:
Why would subnetNameFromURL == "" ever?

Reply:
This is a situation that will happen on existing nodes in a cluster that becomes multi-subnet. These nodes should be treated as being in the default network.
I've found this via manual testing on Basant's prototype and he confirmed it was intended.